decryptor/mysql/types/long.go

```diff
@@ -0,0 +1,98 @@
+package types
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"strconv"
+
+	"github.com/cossacklabs/acra/decryptor/base"
+	"github.com/cossacklabs/acra/decryptor/base/type_awareness"
+	base_mysql "github.com/cossacklabs/acra/decryptor/mysql/base"
+	"github.com/cossacklabs/acra/encryptor/config/common"
+	"github.com/cossacklabs/acra/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+// LongDataTypeEncoder is encoder of TypeLong in MySQL
+type LongDataTypeEncoder struct{}
+
+// Encode implementation of Encode method of DataTypeEncoder interface for TypeLong
+func (t *LongDataTypeEncoder) Encode(ctx context.Context, data []byte, format type_awareness.DataTypeFormat) (context.Context, []byte, error) {
+	strValue := utils.BytesToString(data)
+	intValue, err := strconv.ParseInt(strValue, 10, 64)
```

Comment:
as [I found](https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_binary_resultset.html#sect_protocol_binary_resultset_row), Long type it is 4 byte integer. we can use here 32 bit size instead of 64

Reply:
Agree!